suite/tests/linux/zero-length-mem-ranges.c

```diff
@@ -0,0 +1,40 @@
+#define _GNU_SOURCE
```

Comment:
Please include the copyright + license header

Reply:
Done.